src/ParseUser.js

```diff
@@ -425,6 +425,9 @@ class ParseUser extends ParseObject {
     if (options.hasOwnProperty('installationId')) {
       signupOptions.installationId = options.installationId;
     }
+    if (options.hasOwnProperty('context') && typeof options.context === 'object') {
```

Comment:
`typeof options.context === 'object'` would pass for many types, even `new String()`.

A more specific test would be `Object.prototype.toString.call(context) === '[object Object]'`